packages/xstate-wallet/src/integration-tests/closing.test.ts

```diff
@@ -63,7 +64,7 @@ test('concludes on their turn', async () => {
   await playerA.messagingService.receiveRequest(generateCloseRequest(channelId));
 
   await waitForExpect(async () => {
-    expect(playerA.workflowState).toEqual('done');
+    expect(playerA.workflowState).toEqual('done'); // is this enough of an assertion?
```

Comment:
Doing `ADD_LOGS=true yarn test closing` yields some pretty helpful logs. I convinced myself this test is good, but needs higher coverage.